ansible/roles/sonicv2/templates/etc/systemd/system/swss.j2

```diff
@@ -5,6 +5,8 @@ After=database.service
 
 [Service]
 User={{ sonicadmin_user }}
+# Wait for redis server start before database clean by checking the server listening port 6379
+ExecStartPre=/bin/bash -c "while true; do if [ -n \"$(netstat -l | grep 6379)\" ]; then break; fi; sleep 1; done"
```

Comment:
using "nc -z -w 5 127.0.0.1 6379" to check if the port is open? there could a port 36379 that match your criteria.